os/pm/Kconfig

```diff
@@ -18,6 +17,33 @@ if PM
 config DEBUG_PM
```

Comment:
How about moving this into Kconfig.debug? Because others are there.

Reply:
ok